docs/edgeql/operators/json.rst

```diff
@@ -65,3 +80,57 @@ The contents of JSON *objects* and *arrays* can also be accessed via ``[]``:
     ... })
     ... FILTER j['name'] = <json>'std::bool';
     {{name: 'std::bool', timestamp: '2018-10-18'}}
+
+The element access operator ``[]`` will raise an exception if the
+specified index is not valid or does not exist for the base JSON
+value. It is possible to handle invalid indexes without raising an
+exception with the help of :eql:func:`json_get`.
```

Comment:
"Is is possible" -> on a new line "To access potentially out of bound indexes use the :eql:func:`json_get` function" (for arrays); "To access potentially non-existent fields use the :eql:func:`json_get` function"